doc/classes/XRInterface.xml

```diff
@@ -275,5 +275,14 @@
 		<constant name="XR_ENV_BLEND_MODE_ALPHA_BLEND" value="2" enum="EnvironmentBlendMode">
 			Alpha blend mode. This is typically used for AR or VR devices with passthrough capabilities. The alpha channel controls how much of the passthrough is visible. Alpha of 0.0 means the passthrough is visible and this pixel works in ADDITIVE mode. Alpha of 1.0 means that the passthrough is not visible and this pixel works in OPAQUE mode.
 		</constant>
+		<constant name="XR_VRS_TEXTURE_FORMAT_UNIFIED" value="0" enum="VRSTextureFormat">
+			The texture format is the same as returned by [method XRVRS.make_vrs_texture].
+		</constant>
+		<constant name="XR_VRS_TEXTURE_FORMAT_FRAGMENT_SHADING_RATE" value="1" enum="VRSTextureFormat">
+			The texture format is the same as expected by the Vulkan [code]VK_KHR_fragment_shading_rate[/code] extension.
+		</constant>
+		<constant name="XR_VRS_TEXTURE_FORMAT_FRAGMENT_DENSITY_MAP" value="2" enum="VRSTextureFormat">
+			The texture format is the same as expected by the Vulkan [code]VK_EXT_fragment_shading_rate[/code] extension.
```

Comment:
Is this a typo? The specific `VK_EXT_fragment_shading_rate` doesn't seem to exist.

Reply:
Ack, I already fixed that one, but I guess I forgot to push :-) Really fixed now!